internal/dispatch/graph/check_test.go

```diff
@@ -403,13 +403,13 @@ func TestCheckDebugging(t *testing.T) {
 	}
 }
 
-func newLocalDispatcher(t testing.TB) (context.Context, dispatch.Dispatcher, datastore.Revision) {
+func newLocalDispatcherWithConsistencyLimit(t testing.TB, consistencyLimit uint16) (context.Context, dispatch.Dispatcher, datastore.Revision) {
```

Comment:
I think you meant `concurrencyLimit`
```suggestion
func newLocalDispatcherWithConsistencyLimit(t testing.TB, concurrencyLimit uint16) (context.Context, dispatch.Dispatcher, datastore.Revision) {
```

Reply:
Whoops. Fixed